README.md

```diff
@@ -115,7 +115,32 @@ In order to execute CDS logic, the FHIR Clinical Guidelines Implementation guide
 
 ### Run CQL Tests
 
-To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see documentation for the [CQL Testing Framework](https://github.com/AHRQ-CDS/CQL-Testing-Framework) for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
+To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see the section below, [Setting Up VSAC](#setting-up-vsac), for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
+
+### Setting Up VSAC
+
+Retrieve your VSAC API key from [your UMLS profile](https://uts.nlm.nih.gov/uts/profile) if you already have an account.  If you do not have an account, you must [create one](https://uts.nlm.nih.gov/uts/signup-login) to gain access to an API key.
```

Comment:
I would write out "Value Set Authority Center" the first time VSAC is introduced, kind of like https://github.com/AHRQ-CDS/CQL-Testing-Framework?tab=readme-ov-file#downloading-value-sets does, since users may not be familiar with the acronym.